From 3f0616b15e32def6d01b4535ac0efb51caa07662 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Mon, 2 Dec 2019 18:55:39 +0100
Subject: [PATCH] tests: flowtable: Don't check NFTNL_FLOWTABLE_SIZE

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1717129
Upstream Status: libnftnl commit b2388765e0c44

commit b2388765e0c4405442faa13845419f6a35d0134c
Author: Phil Sutter <phil@nwl.cc>
Date:   Mon Dec 2 18:29:56 2019 +0100

    tests: flowtable: Don't check NFTNL_FLOWTABLE_SIZE

    Marshalling code around that attribute has been dropped by commit
    d1c4b98c733a5 ("flowtable: remove NFTA_FLOWTABLE_SIZE") so it's value is
    lost during the test.

    Assuming that NFTNL_FLOWTABLE_SIZE will receive kernel support at a
    later point, leave the test code in place but just comment it out.

    Fixes: d1c4b98c733a5 ("flowtable: remove NFTA_FLOWTABLE_SIZE")
    Signed-off-by: Phil Sutter <phil@nwl.cc>
    Acked-by: Pablo Neira Ayuso <pablo@netfilter.org>
---
 tests/nft-flowtable-test.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/nft-flowtable-test.c b/tests/nft-flowtable-test.c
index 3edb00d..8ab8d4c 100644
--- a/tests/nft-flowtable-test.c
+++ b/tests/nft-flowtable-test.c
@@ -33,9 +33,11 @@ static void cmp_nftnl_flowtable(struct nftnl_flowtable *a, struct nftnl_flowtabl
 	if (nftnl_flowtable_get_u32(a, NFTNL_FLOWTABLE_USE) !=
 	    nftnl_flowtable_get_u32(b, NFTNL_FLOWTABLE_USE))
 		print_err("Flowtable use mismatches");
+#if 0
 	if (nftnl_flowtable_get_u32(a, NFTNL_FLOWTABLE_SIZE) !=
 	    nftnl_flowtable_get_u32(b, NFTNL_FLOWTABLE_SIZE))
 		print_err("Flowtable size mismatches");
+#endif
 	if (nftnl_flowtable_get_u32(a, NFTNL_FLOWTABLE_FLAGS) !=
 	    nftnl_flowtable_get_u32(b, NFTNL_FLOWTABLE_FLAGS))
 		print_err("Flowtable flags mismatches");
-- 
1.8.3.1

